pkg/expr/classic/classic.go

```diff
@@ -188,11 +188,11 @@ func (ccc *ConditionsCmd) Execute(ctx context.Context, vars mathexp.Vars) (mathe
 func UnmarshalConditionsCmd(rawQuery map[string]interface{}, refID string) (*ConditionsCmd, error) {
 	jsonFromM, err := json.Marshal(rawQuery["conditions"])
 	if err != nil {
-		return nil, fmt.Errorf("failed to remarshal classic condition body: %w", err)
+		return nil, fmt.Errorf("failed to remarshal as classic condition body: %w", err)
```

Comment:
I think the original sentence might have been right because here we are re-marshalling a classic condition into its JSON representation, rather than taking something that isn't a classic condition and making it into the JSON representation of a classic condition?